TCAT/Cells/RouteTableViewCell.swift

```diff
@@ -10,100 +10,52 @@ import FutureNova
 import SwiftyJSON
 import UIKit
 
-protocol TravelDistanceDelegate: NSObjectProtocol {
-    func travelDistanceUpdated(withDistance distance: Double)
+protocol RouteTableViewCellDelegate: class {
+    func updateLiveElements(fun: () -> Void)
+    func getRowNum(for cell: RouteTableViewCell) -> Int?
 }
 
 class RouteTableViewCell: UITableViewCell {
 
     // MARK: Data vars
-    static let identifier: String = "routeCell"
-
-    private var route: Route?
-    private let fileName: String = "RouteTableViewCell"
-    private let networking: Networking = URLSession.shared.request
-
-    // MARK: Log vars
-    private var rowNum: Int?
+    weak var delegate: RouteTableViewCellDelegate?
```

Comment:
can we make private

Reply:
Don't we usually set the delegate from the caller? How would they access?

Reply:
Oh wait wtf I j saw that we pass in the delegate in the configure. Should we be doing it this way, where we pass in a weak delegate to the cell/view, or should we be setting cell.delegate = self in the parent view?

Reply:
i prefer doing it within the configure and making it private b/c no other classes should be accessing the delegate